README.md

```diff
@@ -1,2 +1,111 @@
 # dockstore-cgpbigwig
-Dockstore implementation of cgpBigWig utility
+
+`dockstore-cgpbigwig` provides a suite of commandline tools written in C to manupulate [BigWig] files. These has been packaged specifically for use with the [Dockstore.org](https://dockstore.org/) framework.
+
+[![Join the chat at https://gitter.im/dockstore-cgpbigwig/general][gitter_general]][gitter_badge]
```

Comment:
We have a generic channel for all our `dockstore-cgp*` builds at `https://gitter.im/dockstore-cgp`, it's up to you but it is preferable to have one than several that we have to be signed into as it's likely the bulk of queries will be the same.